frontend/mulighetsrommet-veileder-flate/src/components/modal/Delemodal.tsx

```diff
@@ -0,0 +1,88 @@
+import { Detail, Button, Heading, Modal, Textarea } from '@navikt/ds-react';
+import React, { useState } from 'react';
+import './delemodal.less';
+import { BodyLong } from '@navikt/ds-react';
+import classNames from 'classnames';
+
+interface DelemodalProps {
+  modalOpen: boolean;
+  setModalOpen: () => void;
+  tiltaksgjennomforingsnavn: string;
+  brukerNavn: string;
+  chattekst: string;
+}
+
+const Delemodal = ({ modalOpen, setModalOpen, tiltaksgjennomforingsnavn, brukerNavn, chattekst }: DelemodalProps) => {
+  const startText = chattekst
+    .replace('<Fornavn>', brukerNavn)
+    .replace('<tiltaksnavn>', tiltaksgjennomforingsnavn);
+  const [verdi, setVerdi] = useState(startText);
+  const [meldingSendt, setMeldingSendt] = useState(false);
+
+  const handleSend = () => {
+    setVerdi(startText);
+    setMeldingSendt(true);
+  };
+
+  const clickCancel = () => {
+    setModalOpen();
+    setVerdi(startText);
+    setMeldingSendt(false);
+  };
+
+  const gaTilDialogen = () => {};
```

Comment:
Trengs denne? Eller venter den på videre utvikling når Dialogen er klar for å ta i mot data fra oss?